clients/instance/ibm-pi-workspaces.go

```diff
@@ -38,3 +38,19 @@ func (f *IBMPIWorkspacesClient) Get() (*models.Workspace, error) {
 	}
 	return resp.Payload, nil
 }
+
+// Get all workspaces
+func (f *IBMPIWorkspacesClient) GetAll() (*models.Workspaces, error) {
+	if f.session.IsOnPrem() {
+		return nil, fmt.Errorf("operation not supported in satellite location, check documentation")
+	}
+	params := workspaces.NewV1WorkspacesGetallParams().WithContext(f.ctx).WithTimeout(helpers.PICreateTimeOut)
```

Comment:
Is this the right timeout value? should it be PIGetTimeOut?

Reply:
I have updated it